pkg/cdn/metadata/rdbms/getter.go

```diff
@@ -0,0 +1,13 @@
+package rdbms
+
+import "github.com/gomods/athens/pkg/cdn/metadata/rdbms/models"
+
+// Get retrieves the cdn base URL for a module
+func (s *MetadataStore) Get(module string) (string, error) {
+	result := models.CdnMetadataEntry{}
+	query := s.conn.Where("module = ?", module)
+	if err := query.First(&result); err != nil {
+		return "", err
+	}
+	return result.RedirectURL, nil
```

Comment:
possible to return result.RedirectURL, err? it should not be nil right

Reply:
Do you mean checkin if result == nil? If there is no row matching then First returns an error. Or did you mean cheching RedirectURL?

Reply:
i meant replacing it with single return saying

```go
err := query.First(&result); err != nil {
return result.RedirectURL, nil
```
result should not be nil so result.redirect uri should at least be empty string